lib/buffer.js

```diff
@@ -920,7 +920,7 @@ Buffer.prototype.write = function write(string, offset, length, encoding) {
       length = remaining;
 
     if (string.length > 0 && (length < 0 || offset < 0))
-      throw new ERR_BUFFER_OUT_OF_BOUNDS('length', true);
+      throw new ERR_BUFFER_OUT_OF_BOUNDS();
```

Comment:
Can you explain this and the change to `fs`? I didn't expect to see those here.

Reply:
The `length` was never used in this case. It was a weird API that I fixed.

Originally the implementation was like:

```js
function a (a, b) {
  if (b === true) {
    return 'foo'
  }
  return `${a} foo`
}
```

Now it is:
```js
function a(a = undefined) {
  if (a === undefined) {
    return 'foo'
  }
  return `${a} foo`
}
```